core/src/main/java/com/zextras/carbonio/files/graphql/datafetchers/LinkDataFetcher.java

```diff
@@ -138,7 +138,7 @@ public DataFetcher<CompletableFuture<DataFetcherResult<Map<String, Object>>>> cr
         .has(SharePermission.READ_AND_SHARE)
         && optNode.isPresent() && optNode.get().getNodeType() != NodeType.ROOT
       ) {
-        String publicId = RandomStringUtils.randomAlphanumeric(32);
+        String publicId = RandomStringUtils.randomAlphanumeric(50);
```

Comment:
This method is deprecated. Try to use ` secure()`:

```suggestion
        String publicId = RandomStringUtils.secure().nextAlphanumeric(50);
```